docs/guides/settings_guide.rst

```diff
@@ -347,3 +347,34 @@ Example:
 
 .. _Cesium World Terrain: https://cesium.com/content/cesium-world-terrain/
 .. _Cesium Ion: https://cesium.com/ion/
+
+columnActions
+-------------
+``admin.columnActions``
+
+Column Actions are specific actions that are performed when displaying field data from a feature.
+
+The only action Opensphere currently supports is the ``UrlColumnAction``.  When displaying feature information, this action will scan column names and values via regular expression to create a clickable link.
+
+Example:
+
+.. code-block:: json
+  {
+    "admin": {
+      "columnActions": {
+        "googleMapsAction": {
+          "type": "url",
+          "description": "Create a link to a location on Google Maps.",
+          "regex": {
+            "col": "^DD_LAT_LON$",
+            "val": "[\-\d]\d+\.\d+",
+            "search": "\w",
+            "replace": ","
+          },
+          "action": "https://www.google.com/maps/@%s"
+        }
+      }
+    }
+  }
+
+One of ``"col"`` or ``"val"`` is required. ``"search"`` and ``"replace"`` can be used to transform the text before substituting into the URL, but are entirely optional.
```

Comment:
Thank you for adding to the docs! I would add a performance warning -- since this is run on every field in the column, if the field is huge or the regexes are poorly targeted, it could create a performance sink.

Reply:
Added a paragraph after this line. I hope it reads well enough.